task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -34,19 +37,22 @@ spec:
         #!/usr/bin/env bash
         source /utils.sh
         ### Try to extract binaries with configs > check binaries functionality > check opm validate ###
-        if [ ! -f ../sanity-inspect-image/image_inspect.json ] || [ -z $(cat ../sanity-inspect-image/image_inspect.json) ]; then
+
+        if [[ ! -f ../sanity-inspect-image/image_inspect.json ]] || [[ -z $(cat ../sanity-inspect-image/image_inspect.json) ]]; then
           echo "File ../sanity-inspect-image/image_inspect.json is not generated correctly, please check HACBS_TEST_OUTPUT of task sanity-inspect-image"
           HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -t 'File ../sanity-inspect-image/image_inspect.json is not generated correctly, please check HACBS_TEST_OUTPUT of task sanity-inspect-image!')"
           echo "${HACBS_TEST_OUTPUT}" | tee $(results.HACBS_TEST_OUTPUT.path)
           exit 0
         fi
 
         conffolder=$(cat ../sanity-inspect-image/image_inspect.json | jq -r '.Labels ."operators.operatorframework.io.index.configs.v1"')
+        image_with_digest=$(params.IMAGE_URL)@$(params.IMAGE_DIGEST)
 
-        image_with_digest="$(params.IMAGE_URL)@$(params.IMAGE_DIGEST)"
+        #make temporary folder for image content which will be deleted at the end of this task
+        mkdir to-delete
```

Comment:
Could we name it like catalog-fragment-content to keep semantic of what's inside?

Reply:
I see working dir is a workspace, shouldn't, why do we need copy content to workspace volume? Couldn't we just use local dir or emtpyDir? Copying to shared mount will make it much slower, we don't need it

Reply:
We are using workspace same as from previous step to get: 
../sanity-inspect-image/image_inspect.json

Reply:
yes, for `../sanity-inspect-image/image_inspect.json` it's ok, but why put there some temp data from oc extract?

Reply:
Changed that to /tmp